plugin/storage/grpc/factory.go

```diff
@@ -126,6 +127,9 @@ func (f *Factory) newRemoteStorage(telset component.TelemetrySettings, newClient
 		opts = append(opts, grpc.WithUnaryInterceptor(tenancy.NewClientUnaryInterceptor(tenancyMgr)))
 		opts = append(opts, grpc.WithStreamInterceptor(tenancy.NewClientStreamInterceptor(tenancyMgr)))
 	}
+		
+	opts = append(opts , grpc.WithUnaryInterceptor(bearertoken.NewClientUnaryInterceptor(shared.BearerTokenKey)))
```

Comment:
why does a constant need to be passed as argument? under what conditions a different value would be passed?

Reply:
the token name is constant so i should not pass it as argument